src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/ProviderCourseDeleteController.cs

```diff
@@ -50,6 +50,7 @@ public async Task<IActionResult> GetProviderCourse(int larsCode)
         [HttpPost]
         public async Task<IActionResult> DeleteProviderCourse(ConfirmDeleteStandardViewModel model)
         {
+            TempData.Add(TempDataKeys.DeleteProviderCourseDataKey, true);
```

Comment:
TempData should be set after the mediator call, in case the call fails

Reply:
ok I'll sort that